components/03-modules/slider/slider.hbs

```diff
@@ -0,0 +1,53 @@
+<div class="slider {{ class }}" {{{ attributes }}}>
+    <div class="slider__handler">
+        <div class="slider__frame">
+            <div class="slider__slides">
```

Comment:
Is this nesting required by sliders lib ?

Reply:
only existing in script. Deleted for now and will see what happen